examples/ota-provider-app/ota-provider-common/OTAProviderExample.h

```diff
@@ -21,6 +21,7 @@
 #include <app/CommandHandler.h>
 #include <app/clusters/ota-provider/ota-provider-delegate.h>
 #include <ota-provider-common/BdxOtaSender.h>
+#include <vector>
```

Comment:
Regarding the use of Standard Library in Matter: I think it is likely that the Provider will not be run on a constrained device and so it's OK to use STL. Still, it would be good to double-check this assumption or at least be aware of it.

Reply:
I did think about this when I introduced this vector. I concluded that since this was a "sample reference application" we should be alright. I am presuming customers would evaluate the reference application and augment it for their specific needs.

Reply:
Still, coming from an embedded background, I totally empathize with your statement. I see that quite a few parts of the SDK uses the heap (STL containers, etc.) as well. Would it be more prudent to stick with the data/bss segments, for most if not all of the use-cases?